wire-ios-request-strategy/Sources/Synchronization/Decoding/EventDecoder.swift

```diff
@@ -193,6 +193,10 @@ extension EventDecoder {
     ) async -> [ZMUpdateEvent] {
         let decryptedEvents = await decryptEvent(event: event, publicKeys: publicKeys, proteusService: proteusService)
 
+        guard decryptedEvents.isNonEmpty else {
+            return []
+        }
```

Comment:
fix storing the event id in case of decryption error

Reply:
Why shouldn't we store the event id if there's a decryption error?

Reply:
to avoid duplicate decryption error